tck/src/main/x/tck/union/Basic.x

```diff
@@ -0,0 +1,21 @@
+/**
+ * Very basic union tests.
+ */
+class Basic {
+
+    void run() {
+        basic();
+    }
+
+    // -----------------------------
+    void basic() {
+        typedef (String | Int) as StrInt;
+        StrInt id( StrInt x ) { return x; }
```

Comment:
Not a big thing, but the code style in this line is different from the rest of the tck. I'd prefer a consistent look.

Reply:
I'm not sure what style you are asking for here?

Reply:
Either a long form:

    StrInt id(StrInt x) {
        return x;
    }    
or a short one:

    StrInt id(StrInt x) = x;

Reply:
done